packages/azure-theme/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@fluentui-contrib/azure-theme",
   "version": "0.0.1",
   "dependencies": {
+    "@fluentui/react-icons": ">=2.0.204 <3.0.0",
```

Comment:
had offline convo with @Jacqueline-ms 

in order to start releasing this package:
- remove `private'  field completely
- run `yarn change` to generate changefile 
- merge
- we can release